tools/api-markdown-documenter/src/api-item-transforms/test/Transformation.test.ts

```diff
@@ -637,25 +635,42 @@ describe("ApiItem to Documentation transformation tests", () => {
 						// Body
 						new SectionNode(
 							[
-								new ListNode(
-									[
-										new ListItemNode([
-											{
-												type: "link",
-												url: "/test-package/entry-point-a-entrypoint",
-												children: [{ type: "text", value: "entry-point-a" }],
-											},
-										]),
-										new ListItemNode([
-											{
-												type: "link",
-												url: "/test-package/entry-point-b-entrypoint",
-												children: [{ type: "text", value: "entry-point-b" }],
-											},
-										]),
+								new MarkdownBlockContentNode({
+									type: "list",
+									ordered: false,
+									children: [
+										{
+											type: "listItem",
+											children: [
+												{
+													type: "paragraph",
```

Comment:
Note for reviewers: `mdast` `ListItem`s wrap _block content_ rather than phrasing content. Using a paragraph is the canonical way to include phrasing content in a list item.